webui/react/src/shared/themes.ts

```diff
@@ -152,9 +152,6 @@ const themeBase = {
   statusWarningOn: 'rgba(255, 255, 255, 1.0)',
   statusWarningOnStrong: undefined,
   statusWarningOnWeak: undefined,
-
-  breathButton: 'rgba(143, 143, 143, 0.4)',
-  breathButtonOverlay: 'rgba(143, 143, 143, 0.2)',
```

Comment:
these are used only in `Queue`, so moved them to the `Queue` scss.
@gt2345 is it fine? I can revert it if these could be used in the future

Reply:
That's fine~